BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -256,6 +256,13 @@ public class SendTokenStore: ObservableObject {
       }
     }
   }
+  
+  #if DEBUG
```

Comment:
Why do we have the DEBUG check here? We dont want tests to run except DEBUG or we dont want addresses to setup test addresses on this scheme?

Reply:
only in DEBUG scheme those private properties can be manipulated using this public method.